### ISSUE TITLE
EP-blueprint data not loading properly 

### ISSUE THREAD

**Issue by tenzing-danielm** [elasticpath/react-pwa-reference-storefront#78]
### Step 1: Are you in the right place?

  * For issues or feature requests related to the code **in this repository** file a Github issue.

### Step 2: Describe your environment

  * Device: __N/A___
  * OS version: __N/A___
  * Storefront pulled version: _____
  * Node version: __N/A___
  * Docker version: _N/A____
  * Docker Compose version: _N/A____

### Step 3: Describe the problem:

I've updated the required configuration files, rebuilt the images and redeployed everything. However I'm still seeing mobee data and not vestri data.

Please note: I've deployed a LiveAndAuthor environment. I have an ec2 instance running in a different VPC and whitelisted to connect to EP ELB. 

#### Steps to reproduce:
If I look at extensions/database/ext-
data/src/main/resources/data/liquibase-changelog.xml

this is my file

        

        <include file="ep-blueprint-data/liquibase-changelog.xml"
relativeToChangelogFile="true" />

extensions/database/ext-
data/src/main/resources/environments/local/data-population.properties
liquibase.contexts=default,ep-blueprint

Then I rebuilt package,
Then re-ran the Pipeline for deployment which regenerated everything
including snapshots
I then removed the cloud stack for liveandauthor
Then reran the jenkins liveandauthor

#### Observed Results:

Site is loading with versti content but mobee data

#### Expected Results:

Content / data match

#### Relevant Code:

  ```
  // TODO(you): code here to reproduce the problem
  ```

**Comment by tenzing-danielm** [comment]
some of my stuff got cut.

is in the extensions/database/ext-
data/src/main/resources/data/liquibase-changelog.xml

  <include file="ep-blueprint-data/liquibase-changelog.xml"
relativeToChangelogFile="true" />

**Comment by joel-ep** [comment]
Hi Daniel,

This issue appears to be related to Cloud Ops not publishing the the correct data, rather than an issue with the Reference Storefront itself. Please open a Support Case through the Support Portal for assistance with this.

Thank you,
Joel

**Comment by tenzing-danielm** [comment]
I'm updating the support ticket as well but this is also relevant to the storefront. Apparently CloudOps generates the sample data stuff during the cloud deploy stage.

https://code.elasticpath.com/ep-cloudops/cloud-docs-aws/blob/master/Docs/Data_Population_Config.md

This was my response in the other ticket https://elasticpath.zendesk.com/hc/en-us/requests/1215

Okay so now I think I understand the problem. Looking for a solution.

Yes I made these changes in local. However these changes were done long after CloudOps was executed and a full live/author environment deployed.

Looking at the link you provided it seems to suggest I need to run Cloud-Deploy-aws/Containers/runBootStrap.sh to get sample data uploaded to S3 and into the ext-data-*.jar

How do I run that when I've already deployed an environment.

Given the environment is deployed what is the best method to get this data in ext-data-*.jar?
